src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -159,6 +163,16 @@
                       i18n>Cluster</span>
               </div>
             </div>
+
+            <div class="d-flex flex-column ms-5 me-4 mt-4 mb-4">
```

Comment:
you can use `col` to ensure a proper spacing between `Cluster` and `Hardware` without using spacing. here's how it would look like
```
            <div class="d-flex flex-column ms-4 me-4 mt-4 mb-4">
+            <div class="d-flex flex-column col-md-3 ms-4">
               <ng-template #healthChecks>
                 <ng-container *ngTemplateOutlet="logsLink"></ng-container>
                 <ul>
@@ -164,13 +164,13 @@
               </div>
             </div>
 
-            <div class="d-flex flex-column ms-5 me-4 mt-4 mb-4">
-              <div *ngIf="hardwareEnabled && hardwareError"
+            <div class="d-flex flex-column col-md-3">
+              <div *ngIf="hardwareEnabled && hasHardwareError"
                    class="d-flex flex-row">
                 <i class="text-danger"
                    [ngClass]="[icons.danger, icons.large2x]"></i>
-                <a class="ms-2 mt-n1 lead"
-                   i18n>Hardware</a>
+                <span class="ms-2 mt-n1 lead"
+                   i18n>Hardware</span>
               </div>
             </div>

```